Detectors/Base/include/DetectorsBase/MaterialManager.h

```diff
@@ -47,13 +47,139 @@ class MaterialManager
               Float_t tmaxfd, Float_t stemax, Float_t deemax, Float_t epsil, Float_t stmin, Float_t* ubuf = nullptr,
               Int_t nbuf = 0);
 
+  /// In Geant3/4 there is the possibility to set custom production cuts and to enable/disable certain processes.
+  /// This can be done globally as well as for each medium separately. Hence, for both cases there is one method 
+  /// to set default processes and cuts and another 2 methods to set cuts and processes per medium. In any case, 
+  /// the respective cut/process setting method is a wrapper around a private, more general, method.
+
+  /// Global settings of processes.
+  /// To ignore a certain process to be set explicitly, just set it to o2::Base::MaterialManager::NOPROCESS
+  void Processes( int pair, int comp, int phot, int pfis, int dray, int anni, int brem, int hadr, int munu, 
+                  int dcay, int loss, int muls, int ckov )
```

Comment:
This is an interface with quite many parameters. I am wondering whether this list is fixed. Is it never going to change? (If it changes we would need to make a new API).

It might be worth keeping the old style API
manager->SetProcess("key", value);

Reply:
This is already implemented. Hence, will get rid of these methods with long parameter lists and change this in new PR since they make API static